Examples/Tests/PML/inputs_rz

```diff
@@ -0,0 +1,48 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 500
+amr.n_cell = 32 128
+amr.max_level = 0
+geometry.coord_sys   = 1  # 1: Cylindrical
+geometry.prob_lo     =    0.     -100.e-6
+geometry.prob_hi     =  50.e-6   +100.e-6
+
+warpx.n_rz_azimuthal_modes = 2
+
+#################################
+######## Boundary condition #####
+#################################
+boundary.field_lo = none periodic
+boundary.field_hi = pml  periodic
+
+# PML
+warpx.pml_ncell = 10
+warpx.do_pml_in_domain = 0
+
+#################################
+############ NUMERICS ###########
+#################################
+algo.maxwell_solver = psatd
+warpx.use_filter = 0
+algo.particle_shape = 1
+
+#################################
+############ PARTICLE ###########
+#################################
+particles.species_names = electron
+
+electron.charge = -q_e
+electron.mass = m_e
+electron.injection_style = "singleparticle"
+electron.single_particle_pos = 0. 0. 0.
+electron.single_particle_vel = 10. 0. 0.
+electron.single_particle_weight = 1.
+
+#################################
+########## DIAGNOSTICS ##########
+#################################
+diagnostics.diags_names = diag1
+diag1.intervals = 500
+diag1.fields_to_plot = Ex Ez By
```

Comment:
I can add the extra fields. I had removed them since they had only round off values in them and the values were consistently different between running on my Mac and running in the CI. I'll update the benchmarks using the results from the CI test.

Reply:
I see, I didn't think about that issue. Let me know if that's a persistent issue with this test too. If so, maybe your original code was better, since otherwise this enters the list of RZ tests that are difficult to reset locally. Not sure.

Reply:
I'm seeing that Ey, Bx and Bz are exhibiting on Azure those roundoff issues that you mentioned in your earlier comment. Maybe it's easier to remove them as in your initial version of the input, to avoid future issues.